src/layouts/Splash/styles.js

```diff
@@ -7,6 +7,9 @@ import { createMuiTheme } from '@material-ui/core/styles'
 import variables from 'styles/themes/variables'
 
 export default createMuiTheme({
+  typography: {
```

Comment:
Instead of duplicating the fix in 4 files it is required to extract the fix in one file and import the file in all 4 themes.

Reply:
src/styles/themes/inversed.js
```
/**
 * Copyright 2017–2018, LaborX PTY
 * Licensed under the AGPL Version 3 license.
 */

import { createMuiTheme } from '@material-ui/core/styles'
import variables from './variables'
import config from './fixedThemeConfig'

config.raisedButton = {
  primaryColor: variables.colorAccent2,
  primaryTextColor: variables.colorWhite,
  secondaryColor: variables.colorWhite,
  secondaryTextColor: variables.colorAccent2,
  disabledColor: variables.disabledColor1,
  disabledTextColor: variables.colorWhite,
}

export default createMuiTheme(config)
```

src/styles/themes/fixedThemeConfig.js
```
/**
 * Copyright 2017–2018, LaborX PTY
 * Licensed under the AGPL Version 3 license.
 */

export default {
  typography: {
    useNextVariants: true,
  },
}

```

Reply:
There is not reason for that, because it used only two times and only one param is common. This code will be less readable.

Reply:
I created a task for it (MINT-2089)